src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentsPageV2.jsx

```diff
@@ -49,6 +69,11 @@ function AppointmentsPageV2({
   startNewExpressCareFlow,
   showHomePageRefresh,
 }) {
+  const location = useLocation();
+  const [dropdownValue, setDropdownValue] = useState(
```

Comment:
Does this component re-render after the location changes? If so, I don't think you need component state, I think you could derive the dropdown value from location on each render.

Reply:
Good call.  Removed this and setting the value directly on the dropdown value prop.